.github/workflows/CI.yml

```diff
@@ -7,16 +8,24 @@ on:
   pull_request:
     branches: ["main", "release-*"]
 
+concurrency:
+  group: java-build-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   build:
-    runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+        java: [ 8, 11 ]
```

Comment:
if its easy to add 17 as well, would love to start running builds against it.

Reply:
Sure, just take a look at the output of `11` though. IMHO there's maven warnings because the pom hardcodes to `1.8` which produces some maven warnings. I'll amend with `17` and you tell me whether that's good.

Reply:
yup. those are mostly javadoc errors which have a simple fix based on https://github.com/assertj/assertj/issues/1403. I'll look into fixing it tomorrow (unless you beat me to it).